aws/resource_aws_iam_server_certificate.go

```diff
@@ -180,6 +182,7 @@ func resourceAwsIAMServerCertificateDelete(d *schema.ResourceData, meta interfac
 		if err != nil {
 			if awsErr, ok := err.(awserr.Error); ok {
 				if awsErr.Code() == "DeleteConflict" && strings.Contains(awsErr.Message(), "currently in use by arn") {
+					currentlyInUseBy(awsErr.Message(), meta)
```

Comment:
Nitpick: Do you mind changing the second argument to `*elb.ELB` instead of all `meta`?